cookbooks/hdfsdu/recipes/hdfsdu_deploy.rb

```diff
@@ -0,0 +1,106 @@
+# Cookbook Name : hdfsdu
+# Recipe Name   : hdfsdu_deploy.rb
+# Description   : Download, configure and start hdfsdu
+
+Chef::Resource.send(:include, Hdfsdu::Helper)
```

Comment:
@cbaenziger Thanks for pointing me to [ruby-include-vs-extend](http://www.railstips.org/blog/archives/2009/05/15/include-vs-extend-in-ruby/). Made the change as suggested.